tests/integration-tests/reports/reportsApi-test.js

```diff
@@ -527,18 +606,18 @@ const jobPlatform = process.env.JOB_PLATFORM;
 
                         const getExportedCompareResponse = await reportsRequestCreator.getExportedCompareReport('csv', reportMetaData);
                         expect(getExportedCompareResponse.statusCode).to.be.equal(200);
-                        
+
                         const reportLines = getExportedCompareResponse.text.split('\n');
-                        
+
                         for (let index = 1; index < reportLines.length; index++){
-                            console.log(reportLines[index].split(","));
+                            console.log(reportLines[index].split(','));
```

Comment:
Relevant for all the files - consider if it's worth to do all these lint changes that you did because all the changes of these lines will be on our final commit without something really changed.

Reply:
Still relevant

Reply:
removed